scarb/tests/e2e/fmt.rs

```diff
@@ -0,0 +1,65 @@
+use assert_fs::prelude::*;
+use std::fs;
+
+use crate::support::command::Scarb;
+
+const SIMPLE_ORIGINAL: &str = r"fn main() -> felt { 42 }";
+const SIMPLE_FORMATTED: &str = r"fn main() -> felt {
+    42
+}
+";
+
+#[test]
+fn simple() {
```

Comment:
I would write separate tests for overriding and checking. Consider extracting building of `t` to separate function here.